.clang-format

```
@@ -135,20 +135,20 @@ UseTab: Never
 #IfMacros: []
 IncludeCategories:
   # Local headers (in "") above all else
-  - Regex: '"([A-Za-z0-9.\/-_])+"'
+  - Regex: '"([A-Za-z0-9./_\-])+"'
     Priority: 1
-  # "alpaka/foo.hpp" after local headers (occur inside alpaka)
-  - Regex: '"alpaka/([A-Za-z0-9.\/-_])+"'
+  # "alpaka/foo.hpp" after local headers (occur inside alpaka)  
```

could you remove the extra spaces at the end ?
```suggestion
  # "alpaka/foo.hpp" after local headers (occur inside alpaka)
```